`curate-maintain`: Add contents (#61)
open-education-hub/methodology: chapters/curate-maintain/overview/reading/read.md

# Curating and Maintaining Content

Each topic or class or course will be organized inside a public content repository.

> And, very likely, a private solution repositories.

Reviewer: ```suggestion
And, very likely, a private solution repository.
```

These repositories, which we call origin repositories, are to be forked and used by educators as part of their own needs.

The quality and continued improvement of these origin repositories (public content and private solution) are the main responsibility of **content curators**.

> Each pair of repositories will have one of more content curators, that curating and maintaining content.

Reviewer: ```suggestion
Each pair of repositories will have one of more content curators, that curate and maintain content.
```

Content curators have full permissions on the repositories, they keep contact with educators using the repositories and generally manage the community around them.

In summary, curating and maintaining content implies the following responsibilities:

- Ensure an effective and timely review process for repositories, and quick integration of contributions.
- Oversee content use by educators in delivery repositories.

>   Ask and collect updates from delivery repositories to be upstream in the origin repositories.

Reviewer: ```suggestion
  Ask for and collect updates from delivery repositories to be upstream in the origin repositories.
```
